sdk/core/System.ClientModel/src/Options/ClientPipelineOptions.cs

```diff
@@ -33,7 +40,7 @@ public class ClientPipelineOptions
     /// In most cases, this property will be set to an instance of
     /// <see cref="ClientRetryPolicy"/>.
     /// </remarks>
-    public PipelinePolicy? RetryPolicy
+    public /* virtual */ PipelinePolicy? RetryPolicy
```

Comment:
Making policy properties virtual would mean clients' "client options" subtypes wouldn't need to reallocate to customize, but this makes the setter virtual too.